node/test/unit/tessel.js

```diff
@@ -565,7 +565,20 @@ exports['Tessel.Port.prototype'] = {
     test.done();
   },
 
-  I2C: function(test) {
+  I2CnoArgsAlwaysHasPort: function(test) {
+    test.expect(3);
+
+    var device1 = new this.port.I2C();
```

Comment:
No, `this.port.I2C()` always uses its own port, because it can only have _that_ port. `Tessel.I2C` _will_ fail without a port